deploy/fip_10b.js

```diff
@@ -18,6 +20,8 @@ async function deploy(deployAddress, addresses, logging = false) {
     curveMetapoolAddress, // FEI-3crv metapool
     curve3poolAddress, // Curve 3pool
     '50', // 0.5% max slippage
+    '100000000000000000', // minimum 1:99 FEI in pool
```

Comment:
The addition of max and min multipliers for reporting balances weren't added to deploy